gui/package-lock.json

```diff
@@ -1,8 +1,16269 @@
 {
```

Comment:
Do we need to commit lock file ?

Reply:
not really, I will add this to the  .gitignore file.

Reply:
actually, we should https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5